infrastructure/helm-chart/charts/core/templates/ingress.yaml

```diff
@@ -16,77 +16,14 @@ spec:
                 name: api-websocket
                 port:
                   number: 80
-          - path: /conversations.list
+          - path: /conversations
             pathType: Prefix
             backend:
               service:
                 name: api-communication
                 port:
                   number: 80
-          - path: /conversations.info
```

Comment:
@ljupcovangelski and @pascal-airy I simplified this file a lot by using prefix matching. We don't have to specify every single route if e.g. all `/conversations*` routes go to `api-communication`.